docs/getting_started.md

```diff
@@ -216,6 +218,64 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+If the dataset that you have downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices are stored in an `altExp` named `"adt"` of the `SingleCellExperiment` object in the `processed.rds` file:
+
+```r
+# View the ADT alternative experiment
+altExp(processed_sce)
+
+# You can also explicitly specify the altexp's name:
+altExp(processed_sce, "adt")
+```
+
+The following commands can be used to access the ADT expression matrices:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+It is important to be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
+The `adt_scpca_filter` column indicates which cells should be removed before proceeding with downstream analyses of the ADT data, as determined by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+Specifically, this process identified cells with high levels of ambient concentration and/or, if available, high levels of negative/isotype control ADTs for removal.
+Cells are labeled either as `"Keep"` (cells to retain) or `"Remove"` (cells to filter out).
+
+To filter cells based on this column, use the following command:
+
+```r
+# Filter cells based on ADT QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter == "Keep")]
+```
+
+It is also important to be aware that the normalized ADT expression matrix only contains values for cells labeled as `"Keep"` in the `adt_scpca_filter` column.
+Any cells labeled `"Remove"` have `NA` values in the normalized expression matrix (see {ref}`processed adt data <processing_information:Processed ADT data>` for more details).
+
+
+### Filtering cells based on ADT quality control
+
+In both the `filtered.rds` and `processed.rds` files, quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the alternative experiment's `colData` and can be used for filtering.
+
+To perform the same filtering as we recommended above for the `processed.rds` file, you can use the following command:
```

Comment:
Do we not have the `adt_scpca_filter` column in the `filtered` data too? I thought that we did, and if so we can suggest using that column instead of `discard` which removes an `altExp()` call.

Reply:
Yes, it's there! Will update accordingly, thanks

Reply:
Ah woops, I misread - we do not have this information in `filtered` objects all around.  We only have the `adt_scpca_filter_method`, and similarly for RNA `scpca_filter_method` in the filtered objects. Only the processed objects contain the remove/keep columns.

Reply:
For reference: https://github.com/AlexsLemonade/scpca-nf/blob/5765203dfc65c9ee74c4e02c0b98ea0e45ae7fda/bin/post_process_sce.R#L84-L103

Reply:
They _should_ have those columns!

Added here: 
https://github.com/AlexsLemonade/scpca-nf/blob/3a8d909e06875349f21e102045e33cae7b85d6fc/bin/post_process_sce.R#L98-L103

Then the filtered files are rewritten out: 
https://github.com/AlexsLemonade/scpca-nf/blob/5765203dfc65c9ee74c4e02c0b98ea0e45ae7fda/bin/post_process_sce.R#L237

If they are not appearing in the actual output files, we need to investigate

Reply:
Oh yeah, I see they're saved into `sce`. Hm!  I'll investigate, 50/50 chance i'm just sleepy.

Reply:
It is my distinct pleasure to report back that I am sleepy and everything is there! Did you know that the `metadata` and `colData` slots are not the same slot? This is something I _usually_ know ;) 

Sentence update incoming...